src/components/DashboardResultIndicatorsSection.vue

```diff
@@ -1,21 +1,26 @@
 <template>
   <div class="dashboardResultIndicatorsSection">
     <div class="dashboardResultIndicatorsSection__header">
-      <tabs-list
+      <tab-list
+        aria-label="Velg resultatindikator"
         :tabs="
-          this.resultIndicators.map((resultIndicator) => ({
-            label: resultIndicator.name,
-            tooltip: resultIndicator.description,
+          resultIndicators.map((resultIndicator) => ({
+            tabName: resultIndicator.name,
+            tooltip: {
+              content: resultIndicator.description,
+              placement: 'bottom',
```

Comment:
<img width="428" alt="bilde" src="https://user-images.githubusercontent.com/2866225/190343988-7d30e36f-a091-4fd5-9064-8ebd09d84239.png">
 
Problemet ser ut til å være relatert til overflow på `.tabList`.  Løses tilsynelatende med global config under, eller kanskje tryggest som [config](https://v-tooltip.netlify.app/legacy/v2/#directive) på "directiven" i `TabList.vue`: `..., boundariesElement: 'body',`? Vet ikke akkurat hva som skjer, ikke spesifikt dokumentert, men handler kanskje om "the area that the element will be checked for overflow relative to": https://popper.js.org/docs/v2/utils/detect-overflow/#boundary 😄 
```
Vue.use(VTooltip, {
  defaultBoundariesElement: document.body,
});
```

Reply:
Takk! La til en `boundariesElement: 'body'` på directiven i `TabList`og det ser ut til å funke fint  😊